cmake/googletest-download.cmake

```diff
@@ -0,0 +1,20 @@
+# code copied from https://crascit.com/2015/07/25/cmake-gtest/
```

Comment:
Doesn't seem necessary anymore.  I'm OK for keeping it here for the convenience of others.

Reply:
@trivialfis I added this because this was the easiest way to install Google Test on Windows platform.